pyVor/tests/testPredicates.py

```diff
@@ -122,6 +122,26 @@ def test_incircle_homogeneous(self):
         self.assertEqual(incircle(*inf_triangle[::-1], Point(-1, 1, 1)),
                          -1)
 
+        # Tricky cases:
+        self.assertEqual(incircle(
+            Point(-1, -1, 0), Point(-3, 2, 1), Point(1, 0, 0), Point(0, 1, 0)),
+                         1)
+        self.assertEqual(incircle(
+            Point(-3, 2, 1), Point(-1, -1, 0), Point(1, 0, 0), Point(0, 1, 0)),
+                         -1)
+
+        # The following tests are based on pencil-and-paper drawings
+        # as well as hours spent debugging the DelaunayTriangulation.
+
+        self.assertEqual(incircle(
```

Comment:
This test fails (twice) due to what I assume is the fact that the first point is a point at infinite. For cases like this, does this just reduce to a line-side test where we're checking to see if the 4th point is on the same side of the line formed by the 2nd and 3rd as the 1st point?